data-model/SDP-data-model-templates/codicological-unit_sdp-template.json

```diff
@@ -115,30 +118,48 @@
       "note": [
         ""
       ],
-      "attested_entities": [
-        {
-          "entity_type": "CV",
-          "role": "CV",
-          "id": "URI/ARK",
-          "attested_name_NS": "",
-          "attested_name": "",
-          "note": [
-            "array of strings"
-          ]
-        }
-      ]
-    }
-  ],
-  "associated_entities": [
-    {
-      "entity_type": "CV",
-      "relation": "CV",
-      "id": "URI/ARK",
-      "note": [
-        "array of strings"
-      ]
+      "attested_entities": {
```

Comment:
see comments in manuscript object.

Reply:
oh wait, no this is a bit different. So in texts and codicological units I've kept 'paratext' as a field. Here I've included 'attested' rather than 'associated' entities, and I just updated the field to match with 'associated_entities' so that it has 'attested_person' and 'attested_place'.